### DISCUSSION TITLE
Перестал работать обход

### DISCUSSION THREAD

**Discussion by Quer4ik123** [Flowseal/zapret-discord-youtube#1079] [General]
Не знаю, что писать. Тайтл говорит сам за себя. Примерно неделю назад обход просто перестал работать. Был 1.5.1., сейчас от отчаяния обновил до 1.6.1 - тоже не работает. Файл discord.bat - не работает, general и его альты вообще не запускаются теперь (то есть, запускаются и моментально закрываются). Пробовал сервисы переустанавливать - после этого как раз альты и не работают больше. Очевидно, запускаю от имени администратора, но импакта никакого совсем. А дс мне для хобби очень важен. Что делать?

**Comment by Quer4ik123** [comment]
Добавлю, что при попытке переустановить сервисы именно для discord.bat, мне выдаёт, что везде доступ отказан, хотя запускаю от имени админа

**Reply by Quer4ik123** [reply]
И при переустановке сервиса постоянно писало, что с OpenService ошибка постоянно какая и он не может остановиться

**Reply by medic121** [reply]
+

**Comment by Flowseal** [comment]
https://github.com/Flowseal/zapret-discord-youtube/discussions/252

ютуб работает?

**Reply by amifxf** [reply]
у меня такая проблема работает ютуб но не дискорд(открываю генерал но и все остальное пробывал(провойдер ростелеком если это важно))

**Comment by Paradox0952** [comment]
Та же самая проблема. После обновления винды все обходы такого типа дружно отказали. Сценарий одинаковый - открываю программу, открывается командная строка, закрывается, открывается снова как положено, и после этого, секунды через 2, закрывается снова, с концами.
